test/core-flags/output-flags.test.js

```diff
@@ -3,9 +3,9 @@
 const { run, hyphenToUpperCase } = require('../utils/test-utils');
 const { flagsFromCore } = require('../../packages/webpack-cli/lib/utils/cli-flags');
 
-const outputFlags = flagsFromCore.filter(({ name }) => name.startsWith('output-'));
+const outputFlags = flagsFromCore.filter(({ name }) => name.startsWith('output-') && !name.includes('output-library'));
```

Comment:
Why we exclude `output-library'`? It is just a string

Reply:
I'm adding separate tests for `output-library` related flags, as they are dependent on each other and creates noise with these test cases.

Reply:
I don't understand, developer can use `webpack ./src/index.js --output-library=myLibrary`

Reply:
Actually we are filtering around 20 flags here,
```
output-library
output-library-reset
output-library-amd
output-library-commonjs
output-library-root
output-library-root-reset
output-library-auxiliary-comment
output-library-auxiliary-comment-amd
output-library-auxiliary-comment-commonjs
output-library-auxiliary-comment-commonjs2
output-library-auxiliary-comment-root
output-library-export
output-library-export-reset
output-library-name
output-library-name-reset
output-library-name-amd
output-library-name-commonjs
output-library-name-root
output-library-name-root-reset
output-library-type
output-library-umd-named-define
```

Reply:
Yes, we should fix, `output-library-*` is very popular flags, so we should support them, please focus on this